<launch>
    <arg name="fcu_url" value="/dev/ttyFC0:57600"/>
    <include file="$(find mavros)/launch/px4.launch"/>
    <node name="mavros_interface" pkg="drone_gnc" type="drone_mavros_interface" machine="gnc_machine"/>
</launch>


